search-commons/src/main/java/no/unit/nva/search2/ResourceAwsQuery.java

```diff
@@ -212,8 +208,13 @@ private String expandFields(String value) {
             return ALL.equals(value) || isNull(value)
                 ? ALL
                 : Arrays.stream(value.split(COMMA))
-                .filter(Builder::notInvalid)
```

Comment:
removed static, bad for performance?

Reply:
I wonder about static myself. Does every caller share the static method and have to wait in turn to use it, or is it better because it is loaded only once in memory and every caller can use it in parallell? We have discussed this and we are not really sure